### ISSUE TITLE
Make an explicit "community" section

### ISSUE THREAD

**Issue by samtstern** [firebase/firebaseopensource.com#52]
A section to show off things that are not Google-owned.

**Comment by jhuleatt** [comment]
A start for this may be to move https://github.com/firebase/firebaseopensource.com/blob/master/config/additional_projects.json from an array of strings to an array of objects like:

```json
[
    {
        "project": "angular::angularfire2",
        "googleOwned": true
    },
    {
        "project": "CSFrequency::react-firebase-hooks",
        "googleOwned": false
    }
]
```

to make it easier to know when to render some sort of "Google Official" badge on a repo's page

**Comment by samtstern** [comment]
Yeah for sure, or just two files (one for `additional_google_projects` and one for `additonal_community_projects`)

**Comment by digimbyte** [comment]
if this also extends to communities that specialize in firebase, I would like to put our group forward
[](https://discord.firebase.me/)

**Comment by samtstern** [comment]
@digimbyte we're shifting focus from firebaseopensource.com to a new site, https://devlibrary.withgoogle.com

Right now we're still focused on just GitHub projects and blog posts but maybe one day we'll have a place on the new site to list communities like yours!
